features/bootstrap/VirtualMetricHandleContext.php

```diff
@@ -22,9 +22,11 @@ public function iAddAVirtualMetric()
         $this->page = new MetricsConfigurationListingPage($this);
         $this->assertFind('css', 'a[class="btc bt_success"]')->click();
         $this->page = new MetricsConfigurationPage($this);
+        sleep(70);
```

Comment:
Any reason to add this sleep here, and not on develop branch ?
```suggestion
```

Reply:
@kduret for some reasons the build was not successful and it was displaying "cannot find value test10 in selection list" it only occurs in this branch and when i added this, it worked maybe was just random fail, but i can remove it and retest if possible :) !